binders/kafka-binder/spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/KafkaStreamsBinderUtils.java

```diff
@@ -64,14 +67,25 @@
  * @author Soby Chacko
  * @author Gary Russell
  */
-final class KafkaStreamsBinderUtils {
+public final class KafkaStreamsBinderUtils {
 
 	private static final Log LOGGER = LogFactory.getLog(KafkaStreamsBinderUtils.class);
 
 	private KafkaStreamsBinderUtils() {
 
 	}
 
+	/**
+	 * Utility method to find the method targeted by the key.
+	 *
+	 * @param key name of the method
+	 * @param methods collection of methods to search from
+	 * @return found method as an {@link Optional}
+	 */
+	public static Optional<Method> findTargetMethod(String key, Method[] methods) {
```

Comment:
**[TinyNit]** I think a minor improvement would be to name the method something like `findMethodWithName`.